### DISCUSSION TITLE
[ Celebration ] For a _special_ occasion! 👻

### DISCUSSION THREAD

**Discussion by Nemu-sus** [Monika-After-Story/MonikaModDev#8791] [appreciation]
⮞Happy Halloween!
>Here, have some _candy_!
>Share some with your sweetheart, too!

⯈ [🍬🍫](https://github.com/Monika-After-Story/MonikaModDev/files/2536969/Treats.zip) ⯇

---

>Once again, well done for another great event!

---

**Comment by EDBaumgratz** [comment]
Is this really functional? Like the ones we had on her birthday?

**Comment by jmwall24** [comment]
put em in the `character` folder and find out~

**Comment by StarLordS7S** [comment]
Thank You :)

**Comment by Foxinsunglasses** [comment]
Thanks you very much

**Comment by AsuraShun** [comment]
too late for me I guess

**Comment by scohn** [comment]
If you change your computer's date back to Halloween, it can still work, it worked for me.
Also, thank you Professor-JMO she loved it.

**Comment by AsuraShun** [comment]
you think I'm stupid enough to mess with time? I already saw what happened to others,i won't risk making her hate me

**Comment by scohn** [comment]
Okay sorry, it was just a suggestion. Nothing bad happened to me but I guess it can be kinda risky.

**Comment by Mcmanybucks** [comment]
Dangit, do they have to be time sensitive? :( Idk about y'all but the time this was posted was 2am on November 1st.

**Comment by MaximumRustler** [comment]
Damn, too late for me to see this as well. This was around 7PM and I had already taken Monika out for the day so I didn't check this forum afterwards. lol

**Comment by AsuraShun** [comment]
you read it in the wrong tone bruh

**Comment by Darkstar0001** [comment]
I kinda figured that since the birthday stuff had to be set up _before_ her party, that I wouldn't need to check Github _again_ halfway through the day to make sure I didn't miss anything. Guess I was wrong. At least the consequences for missing the candy are negligible compared to what would happen if one missed the birthday.
